raw_data/preprocessed/africa/zanzibar_english.txt

```diff
@@ -0,0 +1,265 @@
+
+(107) [1] The enforcing institutions under this Act shall be: [a] regarding air and noise: the Ministry responsible for health; [b] regarding quality control of water supplied for drinking and domestic purposes, inland waters and effluents: the Ministry responsible for water resources; [c] regarding solid waste and nonrenewable natural resources: the Ministry responsible for local government; [d] regarding pesticide residues: the Ministry responsible for agriculture and livestock; [e] regarding conservation and sustainable development of fish and marine resources and waters in the coastal areas: the Ministry responsible for fish resources; [f] regarding conservation and sustainable development of forests and wildlife resources: the Ministry responsible for forest resources; [g] all other environmental matters and media: the institution responsible for the environment. [2] The Minister may designate additional enforcing institutions for additional environmental matters by notice in the Official Gazette. [3] Within the above designated sphere of responsibility, enforcing institutions shall: [a] supervises enforcement of National Environmental Standards, orders and notices issued under this Act; [b] verify compliance with this Act through regular reporting, monitoring, sampling, testing and analyses; and [c] provides such assistance as may be required to review an EIS or to assist in an environmental emergency. [4] Each enforcing institution shall have all the powers conferred under sections 23 and 24 to issue stop orders and default notices, revoke permits and confiscate environmental performance bonds. [5] Each enforcing institution shall have all the powers conferred on an environmental officer under section 110 and may delegate in writing its powers to any qualified person whether or not employed by the institution. [6] Each enforcing institution shall cooperate with other enforcing institutions. [7] Each enforcing institution shall keep records of all compliance monitoring, information and environmental data received from monitoring and shall provide the institution responsible for the environment with copies upon request.
```

Comment:
Please replace:
* `[a] supervises enforcement` with `[a] supervise enforcement`; and
* `[c] provides such` with `[c] provide such`.